pkg/validation/validate.go

```diff
@@ -168,6 +174,34 @@ func CompareAssumeRolePolicy(ctx context.Context, request string, target string)
 	return true
 }
 
+//CompareTags compares tags from request and response
+func CompareTags(ctx context.Context, request map[string]string, target []*iam.Tag) bool {
```

Comment:
Does this handle the default tags in the comparison? Default tags(managedBy, Namespace, cluster) is not coming in the request annotation so it is always returns that there is a diff

Reply:
This is checking roleInput tags so default tags are included in request map, unit tests validate the same